Add interfaces/web-transport.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/74301f9/ed/idl/web-transport.idl
Build: https://travis-ci.org/w3c/webref/builds/187361635

stephenmcgruer reviewed (changes requested):
https://github.com/web-platform-tests/wpt/pull/22852 exists for this, but is still blocked on https://github.com/WICG/web-transport/issues/125

autofoolip committed:
Add interfaces/web-transport.idl

Source: https://github.com/w3c/webref/blob/74301f9/ed/idl/web-transport.idl
Build: https://travis-ci.org/w3c/webref/builds/187361635
